Classes/Xclass/DefaultSplitButton.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace T3kit\themeT3kit\Xclass;
+
+use TYPO3\CMS\Backend\Template\Components\Buttons\InputButton;
+use TYPO3\CMS\Backend\Template\Components\Buttons\SplitButton;
+use TYPO3\CMS\Core\Authentication\BackendUserAuthentication;
+
+class DefaultSplitButton extends SplitButton {
+
+    protected $defaultButtonSet = FALSE;
+
+    /**
+     * Returns the current button
+     *
+     * @return array
+     */
+    public function getButton() {
+        if ($this->defaultButtonSet) {
+            return $this->items;
```

Comment:
@yosokus the flow of the code is not very clear. Multiple return statements all over the function. Please follow clean code practices.